### PR TITLE
Add remaining compiler-rt builtin sources. NFC.

### PR DESCRIPTION
**Pull request by sbc100**
This change doesn't actually compile any of them which is
why its NFC.  It just paves the way for compiling them and makes
updating from upstream simple since we just copy everything.

### CODE REVIEW

File: system/lib/update_compiler_rt.py [hunk 1]
```diff
@@ -24,11 +24,9 @@ def main():
...
   os.mkdir(local_builtins)
 
-  listfile = os.path.join(script_dir, 'compiler-rt', 'filelist.txt')
-  for name in open(listfile).read().splitlines():
-    src = os.path.join(upstream_src, name);
-    target = os.path.join(local_builtins, name);
-    shutil.copy2(src, target)
+  for pattern in ('*.c', '*.h', '*.inc'):
+    for src in glob.glob(os.path.join(upstream_src, pattern)):
+      shutil.copy2(src, local_builtins)
```
Comment (tlively) [inline system/lib/update_compiler_rt.py:29]: It looks like this might be partially redundant with the next loop, which could be deleted.

**Comment by kripken** [comment]
Do we expect to need to compile them?

If we don't, I'm not sure it's worth adding them all. It is still equally easy to copy them from upstream in that case - after the copy, the untracked files would be ignored when you commit them.

**Comment by sbc100** [comment]
The plans is to compile them all yes.   I just want to add them first then remove the filelist feature that limits which ones we compile.

There may be one or two we choose to exclude but I see no reason not to compile them all for the wasm backend.  They will only be included on demand anyway.

I'd also like to reduce the differences between upstream and local copies of compiler-rt to make updates easier going forward... so including everything makes thing cleaner in a way.

File: system/lib/update_compiler_rt.py (hunk 1 above)
Comment (sbc100) [reply system/lib/update_compiler_rt.py:29]: Filelist is still used to selectively build only a subset.  Building everything and removing the filelist is a followup to this PR.

Comment (sbc100) [reply system/lib/update_compiler_rt.py]: You are right.. I've got a followup that changes this again.. bit fixed the duplication here now

**Pull request merged**